tests/test_informationmodel.py

```diff
@@ -121,9 +124,14 @@ def test_to_graph_should_return_publisher() -> None:
     @prefix dcat: <http://www.w3.org/ns/dcat#> .
     @prefix foaf:  <http://xmlns.com/foaf/0.1/> .
 
-    <http://example.com/informationmodels/1>  a dcat:Resource ;
-    dct:publisher <https://example.com/organizations/1> ;
-    .
+    <http://example.com/informationmodels/1> a <http://www.w3.org/ns/dcat#Resource> .
+      <http://example.com/informationmodels/1> dct:publisher
+        <https://example.com/organizations/1> ;
+        dct:title "CRD IV - Likviditet NSFR - konsolidert (KRT-1075)"@nb .
+
+    <https://example.com/organizations/1> a <http://xmlns.com/foaf/0.1/Agent> ;
+    dct:identifier "123456789" .
```

Comment:
indent

Reply:
Indent på linje 133, altså?

Reply:
For å vise at dct:identifier tilhører <https://example.com/organizations/1>. Er bare for menneskelesbarhet, ikke noe som kreves av turtle

Reply:
Jepp, skal fikse :+1: